### ISSUE TITLE
VB DataFlowAnalysis.Captured doesn't include range variables captured by a lambda in a query clause

### ISSUE THREAD

**Issue by tmat** [dotnet/roslyn#1312]
Consider the following code:
```VB
Class C
    Function Z(f As Func(Of Integer)) As Integer
        Return 1
    End Function

    Sub F()
        Dim result = From a In {1}
                            From b In {2}
                            Where Z(Function() a + b) > 0
                            Select a
    End Sub
End Class
```

The set of captured variables for the body of method F() doesn't include "a" and "b".
In C# these variables are reported as captured.

**Comment by gafter** [comment]
See also #1291 - C# has the "opposite" problem.

**Comment by gafter** [comment]
@tmat You asked me to tell you when this is fixed. It is fixed.

**Comment by gafter** [comment]
@tmat Can you please "Verify" that this is fixed?
